yojson-bench.opam

```diff
@@ -7,11 +7,12 @@ dev-repo: "git+https://github.com/ocaml-community/yojson.git"
 doc: "https://ocaml-community.github.io/yojson/"
 license: "BSD-3-Clause"
 depends: [
-  "ocaml" {>= "4.04"}
+  "ocaml" {>= "4.08"}
```

Comment:
Why adding this constraint? It is not inherent to `yojson-bench`, and I don't think it should be here (eg if core `0.16` supports `ocaml.4.04` for instance!). `opam` will generate this dependency by transitivity anyway.

Reply:
This is because `core_bench` [supports 4.08 at minimum](https://opam.ocaml.org/packages/core_bench/core_bench.v0.14.0/).

My reasoning here is that if it can't be installed on 4.04 (because even although the code compiles, the dependencies don't) then we can't really test it on such old releases (e.g. via the minimal version mode of OCaml CI) and thus can't make sure our code actually builds on 4.04, the lowest we can check is 4.08.

Reply:
Thanks for the explanation! I would have said that it's not a problem to not be able to test, as anyway it cannot be installed. If at some point `core_bench` required `ocaml >= 4.04`, we would be automatically warned by the CI if it works or not.

But that is very minor thing and I rally easily to your point of view!